frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/reducer.ts

```diff
@@ -38,6 +39,7 @@ export type InternalClusterAction =
   | { type: ActionType.SET_ENCRYPTION; payload: EncryptionType }
   // KMS action
   | { type: ActionType.SET_KMS_ENCRYPTION; payload: KMSConfig }
+  | { type: ActionType.CLEAR_KMS_STATE; payload?: KMSConfig }
```

Comment:
Why payload optional ? Can't we pass empty params to the same action ,  used to set kms `ActionType.SET_KMS_ENCRYPTION` ?